napari/layers/base/base.py

```diff
@@ -984,7 +985,7 @@ def _data_slice(self) -> _ThickNDSlice:
     def _get_ndim(self) -> int:
         raise NotImplementedError
 
-    def _get_base_state(self) -> DeprecatingDict:
+    def _get_base_state(self) -> dict:
```

Comment:
Any objection to making this more specific?

```suggestion
    def _get_base_state(self) -> dict[str, Any]:
```